composer.json

```diff
@@ -20,30 +20,30 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.3",
         "doctrine/common": "^2.3",
         "sonata-project/cache": "^1.0 || ^2.0",
         "sonata-project/doctrine-extensions": "^1.1",
         "sonata-project/form-extensions": "^1.0",
         "sonata-project/twig-extensions": "^1.0",
-        "symfony/asset": "^2.8 || ^3.2 || ^4.0",
-        "symfony/config": "^2.8 || ^3.2 || ^4.0",
-        "symfony/console": "^2.8 || ^3.2 || ^4.0",
-        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
-        "symfony/event-dispatcher": "^2.8 || ^3.2 || ^4.0",
-        "symfony/form": "^2.8 || ^3.2 || ^4.0",
-        "symfony/framework-bundle": "^2.8 || ^3.2 || ^4.0",
-        "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
-        "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
-        "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
-        "symfony/twig-bundle": "^2.8 || ^3.2 || ^4.0",
-        "twig/twig": "^1.34 || ^2.0"
+        "symfony/asset": "^4.3",
+        "symfony/config": "^4.3",
+        "symfony/console": "^4.3",
+        "symfony/dependency-injection": "^4.3",
+        "symfony/event-dispatcher": "^4.3",
+        "symfony/form": "^4.3",
+        "symfony/framework-bundle": "^4.3",
+        "symfony/http-foundation": "^4.3",
+        "symfony/http-kernel": "^4.3",
+        "symfony/options-resolver": "^4.3",
+        "symfony/twig-bundle": "^4.3",
+        "twig/twig": "^2.1"
```

Comment:
Why not 2.0 ?

Reply:
Changed this to 2.11 (latest minor release)